system_monitoring_system/report.py

```diff
@@ -93,7 +93,6 @@ def send_email(email, password, resource='Main Menu'):
         raise AuthenticationError(msg)
 
     session.quit()
```

Comment:
Can you make the functions return the filename? It will be helpful for printing it in the GUI.

Reply:
Which filename?

Reply:
filename = the name of the report with its location, e.g. `~/Downloads/report_2021-04-23_22-43-41.pdf`

Reply:
Done